api_tests/preprints/views/test_preprint_detail.py

```diff
@@ -0,0 +1,271 @@
+from nose.tools import *  # flake8: noqa
+
+from tests.base import ApiTestCase
+from api.base.settings.defaults import API_BASE
+
+from website.files.models.osfstorage import OsfStorageFile
+from tests.factories import PreprintFactory, AuthUserFactory, ProjectFactory, SubjectFactory
+
+
+class TestPreprintDetail(ApiTestCase):
+    def setUp(self):
+        super(TestPreprintDetail, self).setUp()
+
+        self.user = AuthUserFactory()
+        self.preprint = PreprintFactory(creator=self.user)
+        self.url = '/{}preprints/{}/'.format(API_BASE, self.preprint._id)
+        self.res = self.app.get(self.url)
+        self.data = self.res.json['data']
+
+    def test_preprint_detail_success(self):
+        assert_equal(self.res.status_code, 200)
+        assert_equal(self.res.content_type, 'application/vnd.api+json')
+
+    def test_preprint_top_level(self):
+        assert_equal(self.data['type'], 'preprints')
+        assert_equal(self.data['id'], self.preprint._id)
+
+
+def create_file(node, filename):
```

Comment:
This seems like it should go elsewhere, e.g. `utils.py`, and maybe implemented as a `PreprintFileFactory` instead of a helper? Not a blocker.

Also, just discovered that there's already a `create_test_file` helper in `api_tests/utils.py`